processing file asmFiles/beq1.asm (1 of 29)
processing file asmFiles/beq1.asm (1 of 29)
processing file asmFiles/beq2.asm (2 of 29)
processing file asmFiles/bge1.asm (3 of 29)
processing file asmFiles/bge2.asm (4 of 29)
processing file asmFiles/bgeu1.asm (5 of 29)
processing file asmFiles/bgeu2.asm (6 of 29)
processing file asmFiles/blt1.asm (7 of 29)
processing file asmFiles/blt2.asm (8 of 29)
processing file asmFiles/bltu1.asm (9 of 29)
processing file asmFiles/bltu2.asm (10 of 29)
processing file asmFiles/bne1.asm (11 of 29)
processing file asmFiles/bne2.asm (12 of 29)
processing file asmFiles/cfHazard.asm (13 of 29)
processing file asmFiles/count_days.asm (14 of 29)
processing file asmFiles/beq1.asm (1 of 29)
processing file asmFiles/beq2.asm (2 of 29)
processing file asmFiles/bge1.asm (3 of 29)
processing file asmFiles/bge2.asm (4 of 29)
processing file asmFiles/bgeu1.asm (5 of 29)
processing file asmFiles/bgeu2.asm (6 of 29)
processing file asmFiles/blt1.asm (7 of 29)
processing file asmFiles/blt2.asm (8 of 29)
processing file asmFiles/bltu1.asm (9 of 29)
processing file asmFiles/bltu2.asm (10 of 29)
processing file asmFiles/bne1.asm (11 of 29)
processing file asmFiles/bne2.asm (12 of 29)
processing file asmFiles/cfHazard.asm (13 of 29)
processing file asmFiles/count_days.asm (14 of 29)
processing file asmFiles/test.branchjump1.asm (1 of 5)
--- memsim.hex	2025-10-07 19:52:18.840860172 -0400
+++ memcpu.hex	2025-10-07 19:52:18.836860018 -0400
@@ -16,6 +16,8 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
-:0400220000000024B6
-:040024000000003C9C
+:040021000000BA5CC5
+:0400220000000038A2
+:040023000000BA5CC3
+:0400240000000038A0
 :00000001FF
processing file asmFiles/test.ldst1.asm (2 of 5)
--- memsim.hex	2025-10-07 19:52:21.136948568 -0400
+++ memcpu.hex	2025-10-07 19:52:21.132948414 -0400
@@ -15,7 +15,7 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:04004000000000F4C8
+:04004100000000F8C3
+:0400420000000500B5
 :00000001FF
processing file asmFiles/test.ldst2.asm (3 of 5)
--- memsim.hex	2025-10-07 19:52:23.462038080 -0400
+++ memcpu.hex	2025-10-07 19:52:23.457037888 -0400
@@ -26,9 +26,8 @@
 :0400190000D52423C7
 :04001A000075262324
 :04001B00FFFFFFFFE5
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
+:0402000000000F04E7
+:0402010000000F08E2
 :04020300FEEDBEEF5F
 :0403C000000073378F
 :0403C1000000270110
processing file asmFiles/test.loadstore.asm (4 of 5)
--- memsim.hex	2025-10-07 19:52:25.772127015 -0400
+++ memcpu.hex	2025-10-07 19:52:25.767126823 -0400
@@ -14,9 +14,9 @@
 :04000D0000D52423D3
 :04000E0000F52623B0
 :04000F00FFFFFFFFF1
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
+:04002000000000F4E8
+:04002100000000F8E3
+:04002200000000805A
 :04002300DEADBEEFA1
 :04003C000000733716
 :04003D000000270197
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/test.branchjump1.asm (1 of 1)
--- memsim.hex	2025-10-07 20:02:53.105278411 -0400
+++ memcpu.hex	2025-10-07 20:02:53.100278218 -0400
@@ -16,6 +16,6 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
-:0400220000000024B6
-:040024000000003C9C
+:0400220000000038A2
+:0400240000000038A0
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 1)
--- memsim.hex	2025-10-07 20:22:01.307530311 -0400
+++ memcpu.hex	2025-10-07 20:22:01.302530118 -0400
@@ -16,6 +16,6 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
-:0400220000000024B6
-:040024000000003C9C
+:0400220000000038A2
+:0400240000000038A0
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 1)
--- memsim.hex	2025-10-07 20:23:34.060103000 -0400
+++ memcpu.hex	2025-10-07 20:23:34.056102846 -0400
@@ -16,6 +16,6 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
-:0400220000000024B6
-:040024000000003C9C
+:0400220000000038A2
+:0400240000000038A0
 :00000001FF
